ingressPort.tex

```diff
@@ -597,16 +596,33 @@ \section{Data Trace Interface requirements}\label{sec:DataInterfaceRequirements}
 store) using \textbf{itype} 0 except for the final load or store,
 which must retire using the natural \textbf{itype} for the instruction
 (for example, a popret instruction must use \textbf{itype} 13 for the
-final load to signal the return).
+final load to signal the return).  The instruction address reported will be the same for

```

Comment:
Should we update the instruction names to match the ratified mnemonics - `cm.push/cm.pop/cm.popret/cm.popretz`

Reply:
That would make sense - this text was written a long time ago (before push/poop was ratified).  I'll update.